packages/blade/src/components/Typography/Title/Title.tsx

```diff
@@ -0,0 +1,45 @@
+import type { ReactElement } from 'react';
+import type { TextTypes } from '../../../tokens/theme/theme';
+import getPlatformType from '../../../utils/getPlatformType';
```

Comment:
yes it is deprecated, we are not using it for web we just use it to detect for React Native because that's how React Native does it internally and exposes that to us - [Ref code from React Native](https://github.com/facebook/react-native/blob/6b6adcc111/Libraries/Core/setUpNavigator.js)

Reply:
Nice! Funny they decided to go with "ReactNative" instead of "Gecko". One would assume the distinction for the platform being made on the basis of `userAgent` and not on a property that was being demised. 😄